webapp-django/crashstats/settings/local.py-dist

```diff
@@ -168,3 +168,43 @@ SECRET_KEY = 'you must change this'
 #           'component=General&groups=client-services-security',
 #    'label': 'File a bug in bugzilla'
 #}
+
+
+# Dataservice API configuration
+# Extend dataservices settings from settings/base.py here
+# At a minimum, you'll probably want to change db username/password. All
+# dataservice objects inherit resource configuration and so can all
+# have their database resource configuration set once in 'secrets.postgresql'
+# and 'resource.postgresql' keys.
+# Here's an example:
+#from . import base
+#try:
+#    DATASERVICE_CONFIG_BASE = base.DATASERVICE_CONFIG_BASE
+#except AttributeError:
+#    pass
```

Comment:
This doesn't make any sense. 
First of all, we would always have a `base.DATASERVICE_CONFIG_BASE` and if we did get an `AttributeError` it wouldn't later be possible to do `DATASERVICE_CONFIG_BASE.update(`